### ISSUE TITLE
filesystem fragility

### ISSUE THREAD

**Issue by rjkroege** [rjkroege/edwood#29]
I attempted:

```
9p read acme/3/body | head
```

where I believed that window 3 contained more than sufficient text and got this:

```
goroutine 66 [running]:
main.xfidutfread(0xc4203c4000, 0xc420468128, 0xda9, 0xb)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/xfid.go:902 +0x3f5
main.xfidread(0xc4203c4000)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/xfid.go:332 +0x30a
main.xfidctl(0xc4203c4000, 0xc4200ac0c0)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/xfid.go:46 +0x87
created by main.xfidallocthread
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/acme.go:616 +0x1e5
```

**Comment by paul-lalonde** [comment]
Did the file perchance include some unicode characters?  I have a hunch.

**Comment by paul-lalonde** [comment]
Fixed (I think) in 9c7613cca7377a0a4bc933af7ba9522fabc0fb0a
